From 9f8a582bc5fc17fcaf11fb38bdafc20e1a757dd6 Mon Sep 17 00:00:00 2001
From: WangJing <wangjing@mprc.pku.edu.cn>
Date: Mon, 10 Oct 2011 17:14:03 +0800
Subject: [PATCH 014/641] UniCore64:Add definition of SET_PERSONALITY

Fixed error:
fs/binfmt_elf.c:720:2: error: implicit declaration of function 'SET_PERSONALITY'

UniCore64 add it in elf.h,like sh. While UniCore32 add it in elf.c as a function

Signed-off-by: WangJing <wangjing@mprc.pku.edu.cn>
---
 arch/unicore64/include/asm/elf.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/unicore64/include/asm/elf.h b/arch/unicore64/include/asm/elf.h
index 30dcbb9..ccc44ca 100644
--- a/arch/unicore64/include/asm/elf.h
+++ b/arch/unicore64/include/asm/elf.h
@@ -32,4 +32,6 @@ typedef struct fpu_regs elf_fpregset_t;
 	((x)->e_ident[EI_CLASS] == ELF_CLASS && \
 	(x)->e_machine == EM_UNICORE64)
 
+#define SET_PERSONALITY(ex) set_personality(PER_LINUX)
+
 #endif /* __ASM_UNICORE64_ELF_H__ */
-- 
1.7.9.5

